spec.bs

```diff
@@ -3054,40 +3057,67 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   : <dfn>is chaff</dfn>
   :: A [=boolean=] field indicating whether this response should be ignored.
   : <dfn>ad render url</dfn>
-  :: The [=URL=] of the ad for the winning bid.
+  :: [=URL=]. The [=leading bid info/leading bid=]'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=] from the
+     auction executed on the trusted auction server.
   : <dfn>ad components</dfn>
-  :: A [=list=] of [=URLs=] containing the ad components for the winning bid.
+  :: A [=list=] of [=URLs=]. A list of [=leading bid info/leading bid=]'s
+     [=generated bid/ad component descriptors=]'s [=ad descriptor/urls=] from the
+     auction executed on the trusted auction server.
   : <dfn>interest group name</dfn>
-  :: The winning interest group's name as a [=string=].
+  :: The [=leading bid info/leading bid=]'s [=generated bid/interest group=]'s
+     [=interest group/name=] as a [=string=] from the
+     auction executed on the trusted auction server.
   : <dfn>interest group owner</dfn>
-  :: The winning interest group's owner [=origin=].
+  :: An [=origin=]. The [=leading bid info/leading bid=]'s
+     [=generated bid/interest group=]'s [=interest group/owner=] from the
+     auction executed on the trusted auction server.
   : <dfn>bidding groups</dfn>
-  :: A [=list=] of interest group owner and interest group name pairs that bid
-     in this auction.
+  :: A [=list=] of interest group [=interest group/owner=] and
+     interest group [=interest group/name=] pairs that bid
+     in the auction executed on the trusted auction server.
   : <dfn>score</dfn>
-  :: {{double}} or Null. TODO
+  :: Null or {{double}}. Null when the server auction is not a component auction.
+     For component auctions, contains the [=leading bid info/top score=]
+     from the component auction executed on the trusted auction server.

```

Comment:
probably better to link to  {{ScoreAdOutput/desirability}.
```suggestion
  :: Null or {{double}}. Null if the server auction is not a component auction,
     otherwise the {{ScoreAdOutput/desirability}} of component auction's winning bid.
```